2021-finalproject/TNTksals/scripts/check-environment.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
This script will be better if print detail information after a test failed

Reply:
> This script will be better if print detail information after a test failed

I have added the information about the failed test. Anything else you would like me to do? In addition, I don't know why I dismissed your stale review. I am sorry. Could you reviewe me again?

Reply:
> I have added the information about the failed test. Anything else you would like me to do? In addition, I don't know why I dismissed your stale review. I am sorry. Could you reviewe me again?

Don't worry, your new commit will dismiss review automatically